superset-frontend/plugins/plugin-chart-echarts/src/Sunburst/types.ts

```diff
@@ -52,6 +52,10 @@ export const DEFAULT_FORM_DATA: Partial<EchartsSunburstFormData> = {
   labelType: EchartsSunburstLabelType.Key,
   showLabels: false,
   dateFormat: 'smart_date',
+  innerRadius: 30,
+  outerRadius: 70,
+  donut: true,
+  showNulls: true,
```

Comment:
This will change how all current sunburst charts are rendered. While I'm not strongly against/for this, I'm not sure we want to forcefully change all current charts. For this reason I would recommend changing the defaults to align with the current behavior, and defer changing the defaults to a later date when there's been a decision to to change the defaults. Maybe we could discuss changing the defaults for the pie chart and sunburst chart in a forthcoming town hall? CC @rusackas @michael-s-molina

Reply:
Default values were set with the same values that were hardcoded (innerRadius = 30, outerRadius = 70) or to keep current default behavior (donut = true, showNulls = true).